### PR TITLE
Azure pro cvm noble jammy

### PR DESCRIPTION
**Pull request by jrtknauer**
Publication of Azure Pro CVM for Jammy and Noble has been completed.

### CODE REVIEW

File: azure/azure-how-to/instances/find-ubuntu-images.rst [hunk 1]
```diff
@@ -112,6 +112,9 @@ Ubuntu Pro 24.04 LTS - Noble Numbat
    * - AMD64
      - Gen1
      - ``Canonical:ubuntu-24_04-lts:ubuntu-pro-gen1:latest``
+   * - AMD64
+     - Gen2
+     - ``Canonical:ubuntu-24_04-lts:ubuntu-pro-cvm:latest``
```
Comment (k-dimple) [inline azure/azure-how-to/instances/find-ubuntu-images.rst:117]: I am not sure if we need to add this here (in the Pro offers section), as we have a separate section for CVM offers. Maybe it makes sense to add it to both.

File: azure/azure-how-to/instances/find-ubuntu-images.rst [hunk 2]
```diff
@@ -213,6 +219,22 @@ Ubuntu CVM 22.04 LTS - Jammy Jellyfish
      - Gen2
      - ``Canonical:0001-com-ubuntu-confidential-vm-jammy:22_04-lts-cvm:latest``
 
+Ubuntu Pro CVM 22.04 LTS - Jammy Jellyfish
```
Comment (k-dimple) [inline azure/azure-how-to/instances/find-ubuntu-images.rst:222]: Do we really need this separate section? Couldn't we just add this one to the table above (in the one for 'Ubuntu CVM 22.04 LTS - Jammy Jellyfish') - like you did for Noble?

File: azure/azure-how-to/instances/find-ubuntu-images.rst (hunk 1 above)
Comment (jrtknauer) [reply azure/azure-how-to/instances/find-ubuntu-images.rst:117]: This is the crux of the issue that I have raised with alliances in how we have canonically "described" our Pro offerings. We are now rapidly generating Pro flavors of a range of base products. It used to simply just be only "Ubuntu Pro" - implicit as Pro Server - and a small set of extraneous Pro-only products (FIPS and CIS). But now we are producing (and promoting in many cases) the Pro variant over the base product.

File: azure/azure-how-to/instances/find-ubuntu-images.rst (hunk 2 above)
Comment (jrtknauer) [reply azure/azure-how-to/instances/find-ubuntu-images.rst:222]: Unlike the Noble, the Jammy Pro CVM offer is a standalone offer. It cannot be sensibly grouped in under the `Ubuntu CVM 22.04 LTS - Jammy Jellyfish` section as it belongs to a completely separate offer with its own links (and as an Azure Virtual Machine is not available in China, so the quickstart link there does not apply).

This is another issue that has been raised with Alliances. Realistically we should not have even produced Pro CVM for Jammy, but we agreed to proceed with it out of Madrid so we're left with another edge case to work around.

**Pull request merged**